src/systems/eu/implementations/react/packages/typography/stories/Index.jsx

```diff
@@ -1,20 +1,41 @@
 /* eslint-disable import/no-extraneous-dependencies */
-import React from 'react';
+import React, { Fragment } from 'react';
 import { storiesOf } from '@storybook/react';
-import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+import { withKnobs, text } from '@storybook/addon-knobs';
 
+import demoContentHeading from '@ecl/eu-specs-typography/demo/data--heading';
 import demoContentParagraph from '@ecl/eu-specs-typography/demo/data--paragraph';
 
 storiesOf('Typography', module)
```

Comment:
can you prefix it with "Utilities"? i.e. `storiesOf('Utilities/Typography', module)`
it makes it easier to find stories (note: we should do it for other stories as well)